pkg/operator/controller/ingress/deployment.go

```diff
@@ -436,16 +436,28 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 
 	routeAdmission := operatorv1.RouteAdmissionPolicy{
 		NamespaceOwnership: operatorv1.StrictNamespaceOwnershipCheck,
+		WildcardPolicy:     operatorv1.WildcardPolicyDisallowed,
 	}
-	if admission := ci.Spec.RouteAdmission; admission != nil && len(admission.NamespaceOwnership) > 0 {
-		routeAdmission.NamespaceOwnership = ci.Spec.RouteAdmission.NamespaceOwnership
+	if admission := ci.Spec.RouteAdmission; admission != nil {
+		if len(admission.NamespaceOwnership) > 0 {
+			routeAdmission.NamespaceOwnership = ci.Spec.RouteAdmission.NamespaceOwnership
+		}
+		if len(admission.WildcardPolicy) > 0 {
+			routeAdmission.WildcardPolicy = ci.Spec.RouteAdmission.WildcardPolicy
+		}
 	}
 	switch routeAdmission.NamespaceOwnership {
 	case operatorv1.StrictNamespaceOwnershipCheck:
 		env = append(env, corev1.EnvVar{Name: "ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK", Value: "false"})
 	case operatorv1.InterNamespaceAllowedOwnershipCheck:
 		env = append(env, corev1.EnvVar{Name: "ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK", Value: "true"})
 	}
+	switch routeAdmission.WildcardPolicy {
+	case operatorv1.WildcardPolicyDisallowed:
+		env = append(env, corev1.EnvVar{Name: "ROUTER_ALLOW_WILDCARD_ROUTES", Value: "false"})
+	case operatorv1.WildcardPolicyAllowed:
+		env = append(env, corev1.EnvVar{Name: "ROUTER_ALLOW_WILDCARD_ROUTES", Value: "true"})
+	}
```

Comment:
Just FYI, during the namespace ownership implementation we noted that this approach will cause a rollout even though for default installs the setting hasn't changed, but we're okay with it (https://github.com/openshift/cluster-ingress-operator/pull/345#discussion_r366545076)

Reply:
Thanks for the heads-up.